src/main/java/euler2/EulerTwo.java

```diff
@@ -0,0 +1,22 @@
+package euler2;
+
+public class EulerTwo {
+    public int sumFibonacciEvenNumbers(int limit) {
```

Comment:
Hej - jest już dużo lepiej, ale dało się to zapisać jeszcze prościej - bez zmiennej fibonacci3:
```
        int sumOfPositiveNumbers = 0;
        int currentNumber = 1;
        int previousNumber = 1;

        while (currentNumber <= maxNumber) {
            if (currentNumber % 2 == 0) {
                sumOfPositiveNumbers += currentNumber;
            }

            int tmp = previousNumber;
            previousNumber = currentNumber;
            currentNumber += tmp;
        }

        return sumOfPositiveNumbers;
```

Reply:
Dzięki za sugestię, nie wpadłabym na takie rozwiązanie :) Dobrze rozumiem że zmienna maxNumber to nasz limit?

Reply:
Tak dokładnie - `maxNumber` jest to limit a `tmp` to tymczasowa zmienna która pozwala nam na przeskoczenie o 1 miejsce wzwyż.